### PR TITLE
Home task: HTML 01

### PR DESCRIPTION
**Pull request by DashikS**
All tasks're completed

### CODE REVIEW

File: dsokolova/01_HTML/additional.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+		<h3>Тег address</h3>
+		<address>Kuprevicha, 1/1, 220141, Minsk, Belarus. Phone: +375.17.2686911</address>
+		<hr>
+		<h3>Семантические и функциональные строчные теги</h3>
+		<p>Пожелания автора:
+		<s><font color="red">Данный текст убрать из монографии</font></s> и заменить его на <u><font color="blue">вновь помещённый текст</font></u>.</p>
+		<hr>
+		<h3>Работа со строчными и блоковыми цитатами</h3>
+		<p>Цитата короткая:<br>
+		<i>И всё­таки она вертится!</i></p>
```
Comment (pluhin) [inline dsokolova/01_HTML/additional.html]: Here need or cite or q tag. This is real quote

File: dsokolova/01_HTML/python_01.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+		строка и пойдём дальше.</p>
+		<p>Введите эти 4 строки:</p>
+<pre>#!/usr/bin/env python
+print 'Hello. I am a python program.'
+name = raw_input("What is your name?")
+print "Hello there, " + name + "!"</pre>
+		<p>Это всё. Сохраните файл как <strong>hello.py</strong> в любом месте. Я предлагаю сохранить его в папке
+		<b>python_examples</b> в домашнем каталоге. Этот пример показывает, как просто написать программу на
+		Python. Для запуска программы нужно сделать её файл исполняемым. Для этого введите в терминале</p>
+		<p><code><strong><i><u>chmod +x hello.py</u></i></strong></code></p>
```
Comment (pluhin) [inline dsokolova/01_HTML/python_01.html]: Use tag p just for \"human\" text

**Comment by kiradzi** [comment]
i am in trouble.... 
remote: Permission to pluhin31/sc.it-academy.by.git denied to kiradzi.
fatal: unable to access 'https://github.com/pluhin31/sc.it-academy.by.git/': The requested URL returned error: 403
----how can I get a permission?????

**Comment by pluhin** [comment]
> i am in trouble....
> remote: Permission to pluhin31/sc.it-academy.by.git denied to kiradzi.
> fatal: unable to access 'https://github.com/pluhin31/sc.it-academy.by.git/': The requested URL returned error: 403
> ----how can I get a permission?????

For next commits use your own repository

**Comment by kiradzi** [comment]
ok

**Pull request merged**